governance-exercises/JanSky Polling Exercise 4 - Deploy Emergency Fix A in imminent executive -  August 9, 2023.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Deploy Emergency Fix A in imminent executive -  August 9, 2023
+summary: Signal your support or opposition to deploying Fix A (an immediate but costly remedy) in reaction to a verified bug.
+discussion_link: $discussion_link
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2023-08-09T18:00:00
+end_date: 2023-08-10T06:00:00
+---
+# Poll: Deploy Emergency Fix A in imminent executive -  August 9, 2023
+
+The Governance Facilitator has placed a Governance Poll into the voting system on behalf of the Maker Community. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for **12 hours** beginning on Wednesday, August 9 at 18:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following emergency response:
+* Deploy Fix A which "prevents the high-severity exploit immediately upon the executive vote passing" and "is likely to incur losses to MKR holders of approximately 20 million DAI over two weeks."
+
+Note that Fix B will "closes the high-severity exploit approximately two weeks after the executive vote passes" at "no risk of incurring financial loss to MKR holders" and could be deployed independently from Fix A.
+
+Due to the potential for a griefing attack, we cannot disclose more information on the nature of the bug discovered. Fix A is the recommended course of action by the team deploying the executive spell, but given the high cost and an alternative recommendation, we request urgent guidance from MKR voters.
+
+Please review the discussion [thread]($discussion_link) to help inform your position before voting.
+
+## Outcomes
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option then the following actions will be taken:**
+* Fix A will be included in an imminent Executive Vote.
+* If the Executive Vote passes, then these changes will become active in the Maker Protocol after the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay) has expired.
+
+**If the votes for the 'No' option equal or exceed the votes for the 'Yes' option then no further action will be taken without a recommendation from the Protocol Scope Facilitator at this time.**
```

Comment:
Kind of. Regardless of the outcome here, Fix B will be deployed in an executive asap. It's not strictly due to the outcome of this poll, but given the situation it's worth highlighting that _something_ will still happen if this is voted down.